Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/util/factory.go

```diff
@@ -1053,7 +1053,19 @@ func (f *Factory) PrinterForMapping(cmd *cobra.Command, mapping *meta.RESTMappin
 		if err != nil {
 			columnLabel = []string{}
 		}
-		printer, err = f.Printer(mapping, GetFlagBool(cmd, "no-headers"), withNamespace, GetWideFlag(cmd), GetFlagBool(cmd, "show-all"), GetFlagBool(cmd, "show-labels"), isWatch(cmd), columnLabel)
+
+		printOptions := kubectl.PrintOptions{
+			NoHeaders:          GetFlagBool(cmd, "no-headers"),
+			WithNamespace:      withNamespace,
+			WithKind:           false,
+			KindName:           "",
```

Comment:
You can omit `WithKind` and `KindName`, since then are always the zero value.

Isn't it valid to use the "show-kind" flag here?